src/scenario.jl

```diff
@@ -1002,7 +1041,7 @@ function run_model(
         # Calculates scope for coral fedundity for each size class and at each location
         fecundity_scope!(fec_scope, fec_all, fec_params_per_m², C_cover_t, habitable_areas)
 
-        loc_coral_cover = _loc_coral_cover(C_cover_t)
+        loc_coral_cover = loc_coral_cover(C_cover_t)
```

Comment:
Won't this replace the method or is this a non-issue?

I can't remember if Julia can distinguish between a variable and a function name...